gradoop-common/src/test/java/org/gradoop/common/model/impl/pojo/ElementTest.java

```diff
@@ -0,0 +1,133 @@
+package org.gradoop.common.model.impl.pojo;
+
+import org.gradoop.common.model.impl.id.GradoopId;
+import org.gradoop.common.model.impl.properties.Properties;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.CALLS_REAL_METHODS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.withSettings;
+
+public class ElementTest {
+
+  @Test
+  public void testSetId() {
+    Element elementMock = mock(Element.class, CALLS_REAL_METHODS);
+    GradoopId id = GradoopId.get();
+    elementMock.setId(id);
+
+    assertSame(id, elementMock.getId());
+  }
+
+  @Test
+  public void testSetProperty() {
+    Element elementMock = mock(Element.class, CALLS_REAL_METHODS);
+    elementMock.setProperty("key", "");
```

Comment:
You can use constants of `GradoopTestUtils` instead of inline values (like `KEY_0` instead of `"key"`).